cubedash/index/postgis/_api.py

```diff
@@ -377,6 +377,90 @@ def product_time_summary(self, product_id: int, start_day, period):
                 )
             )
 
+    @override
+    def collection_cols(self):
+        """Get all columns necessary for creating a Collection"""
```

Comment:
Can't the documentation string be put on the abstract method instead, and all implementing classes automatically gets the documentation string? (That is how my PyCharm behaves with the documentation in some other code base, and I haven't changed any configuration.)

Reply:
Documentation and type annotation improvements need to be done not just for what I've implemented in this PR but for all other methods as well. I think that should be handled in a separate PR.